accord-core/src/test/java/accord/utils/Gens.java

```diff
@@ -262,6 +266,56 @@ private static int index(RandomSource rs, float[] weights)
         return i;
     }
 
+    public static Gen<Gen.IntGen> mixedDistribution(int minInclusive, int maxExclusive, int numBuckets)
```

Comment:
creates uniform buckets within the range then each invoke selects from the index using a chosen distro, and within this range we use another distro